Classes/PullRequestReviews/PullRequestReviewCommentsViewController.swift

```diff
@@ -164,11 +168,14 @@ final class PullRequestReviewCommentsViewController: MessageViewController,
 
     func listAdapter(_ listAdapter: ListAdapter, sectionControllerFor object: Any) -> ListSectionController {
         switch object {
-        case is NSAttributedStringSizing: return IssueTitleSectionController()
-        case is IssueCommentModel: return IssueCommentSectionController(
-            model: self.model,
-            client: client,
-            autocomplete: autocomplete
+        case is NSAttributedStringSizing:
+            return IssueTitleSectionController()
+        case is IssueCommentModel:
+            return IssueCommentSectionController(
+                model: model,
+                client: client,
+                autocomplete: autocomplete,
+                issueCommentDelegate: self
```

Comment:
IMO it's better to separate random style updates into their own PR. Otherwise it makes reviewing these more difficult (not sure what is new code vs just style changes).

Reply:
I don't understand this! 🙃

Reply:
He means the changes in whitespace and newlines.